Ports/libyaml/package.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port="libyaml"
+version="0.2.5"
+files="http://pyyaml.org/download/libyaml/yaml-${version}.tar.gz yaml-${version}.tar.gz fa240dbf262be053f3898006d502d514936c818e422afdcf33921c63bed9bf2e"
```

Comment:
The canonical source for libyaml is here

https://github.com/yaml/libyaml

Reply:
With the release link being this one https://github.com/yaml/libyaml/releases/download/0.2.5/yaml-0.2.5.tar.gz

Reply:
Ok, thank you